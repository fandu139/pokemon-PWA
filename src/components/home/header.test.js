import React from "react";
import Render from 'react-test-renderer';
import Header from './header';

describe('Home Pokemon', () => {
  const props = {
    logo : '',
    title: 'Pokemon Home'
  }
  test('Home Header', () => {
    const component = Render.create(<Header {...props} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
