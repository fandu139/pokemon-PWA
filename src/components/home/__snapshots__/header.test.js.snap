// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Pokemon Header 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src=""
  />
  <h1
    className="App-title"
  >
    
  </h1>
</header>
`;

exports[`Home Pokemon Home Header 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src=""
  />
  <h1
    className="App-title"
  >
    Pokemon Home
  </h1>
</header>
`;

exports[`Home Pokemon Home Loading 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
  />
  <h1
    className="App-title"
  />
</header>
`;
